chia/_tests/wallet/sync/test_wallet_sync.py

```diff
@@ -536,7 +537,13 @@ async def test_request_additions_errors(simulator_and_wallet: OldSimulatorsAndWa
     assert response.proofs is not None
     assert len(response.proofs) == 1
     assert len(response.coins) == 1
-
+    root = (await full_node_api.full_node.block_store.get_full_block(last_block.header_hash)).foliage_transaction_block.additions_root
+    assert confirm_not_included_already_hashed(root, response.proofs[0][0], response.proofs[0][1])
```

Comment:
It's not obvious why we would expect this coin to not exist in this block. Do we not also request an addition that actually does?